client/trino-cli/src/test/java/io/trino/cli/TestClientOptions.java

```diff
@@ -162,15 +161,14 @@ public void testURLHostOnly()
 
     @Test
     public void testURLParams()
-            throws SQLException
     {
-        Console console = createConsole("trino://server.example:8080/my-catalog/my-schema?source=my-client");
```

Comment:
This whole test is for checking params, so removing this invalidates the test (maybe it duplicates another test?).

Reply:
I reverted this change.